### ISSUE TITLE
Error when opening loader

### ISSUE THREAD

**Issue by never147** [shupershuff/Diablo2RLoader#15]
This doesn't appear to affect the usability of the loader (for me) but I thought I'd log a report so you can assess. See attached image and video for error. It happens before the refresh to get into the menu.
[](url)
![Screenshot 2023-08-28 180135](https://github.com/shupershuff/Diablo2RLoader/assets/1729544/dea967ab-6c82-411f-9b95-75b2cee3f9fc)

https://github.com/shupershuff/Diablo2RLoader/assets/1729544/4fc436eb-bf5b-45a4-9b70-c35ee020221c

**Comment by shupershuff** [comment]
Yea I noticed that at the start of the month. Turns out that the date field didn't convert properly if the last recorded date was a single digit. As such, the issue only presented itself if the script was opened on the 1st of the month through to the 9th. Frustrating as there will be some people who don't notice and miss future updates. That said I don't have any updates planned at this time.

This issue has been fixed in v1.8.3, but one of two things is required to fix it. Either:
- Manually update the D2Loader.ps1 file by downloading the latest release and copying D2Loader.ps1 from the new release to overwrite your current D2Loader.ps1 file.
- Alternatively, open the stats.csv file, and delete the value for "LastUpdateCheck".
